src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,59 @@
-export const Tabs = () => {};
+/* eslint-disable no-console */
+import { useEffect } from 'react';
+import { Tab } from '../../types/tab';
+
+type Props = {
+  tabs: Array<Tab>,
+  selectedTab: Tab,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs = ({
+  tabs, selectedTab, onTabSelected,
+}: Props) => {
+  useEffect(() => {
+    if (!tabs.some(t => t === selectedTab)) {
+      onTabSelected(tabs[0]);
+    }
+  });
```

Comment:
I don't really know what's the point of that effect 🤔
could you elaborate @anpiet? 👀